### ISSUE TITLE
Separate C++/Git into separate tutorial modules, add GitHub workflow to git portion

### ISSUE THREAD

**Issue by TrevorEdwards** [cornell-cup/cs-fruits#1]
Mixing C++ with git causes people following the tutorial live to encounter a lot of issues. While this tutorial may be successful in a one-on-one setting or done offline, it will not do well for a large-group tutorial due to this issue.

**Comment by ScottSWu** [comment]
I don't think it was mixing both, I think it was just the C++ part that lost some people. But yes, this should be done is a "workshop" style meeting, where people can still ask questions, but otherwise it's just on their own.
